Zilla crashes with `IllegalArgumentException: cannot accept missingValue` when using `defaultOffset: live`

<!--
  - Please search to see if an issue already exists for the issue you encountered. Duplicate issues will be closed. [Current GitHub Issues](https://github.com/aklivity/zilla/issues)
  - Please make sure your Zilla version is up to date. [Latest Zilla Release](https://github.com/aklivity/zilla/releases/latest)
-->

**Describe the bug**
Zilla crashes with `IllegalArgumentException: cannot accept missingValue` when using `defaultOffset: live` in the `sse.kafka.fanout` zilla example

**To Reproduce**
Steps to reproduce the behavior:
1. Using the [sse.kafka.fanout](https://github.com/aklivity/zilla-examples/tree/main/sse.kafka.fanout) example
2. add the `defaultOffset: live` config option to the kafka `cache_server`
```yaml
      topics:
        - name: events
          defaultOffset: live
```
3. start the zilla exmple
4. Zilla crashes with the below error

```
org.agrona.concurrent.AgentTerminationException: java.lang.IllegalArgumentException: cannot accept missingValue
    at io.aklivity.zilla.runtime.engine@0.9.80/io.aklivity.zilla.runtime.engine.internal.registry.EngineWorker.doWork(EngineWorker.java:823)
    at org.agrona.core/org.agrona.concurrent.AgentRunner.doWork(AgentRunner.java:304)
    at org.agrona.core/org.agrona.concurrent.AgentRunner.workLoop(AgentRunner.java:296)
    at org.agrona.core/org.agrona.concurrent.AgentRunner.run(AgentRunner.java:162)
    at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: java.lang.IllegalArgumentException: cannot accept missingValue
    at org.agrona.core/org.agrona.collections.Long2LongHashMap.putIfAbsent(Long2LongHashMap.java:271)
    at io.aklivity.zilla.runtime.binding.kafka@0.9.80/io.aklivity.zilla.runtime.binding.kafka.internal.stream.KafkaCacheBootstrapFactory$KafkaBootstrapStream.onPartitionLeaderReady(KafkaCacheBootstrapFactory.java:784)
    at io.aklivity.zilla.runtime.binding.kafka@0.9.80/io.aklivity.zilla.runtime.binding.kafka.internal.stream.KafkaCacheBootstrapFactory$KafkaBootstrapFetchStream.onFetchReplyBegin(KafkaCacheBootstrapFactory.java:1548)
    at io.aklivity.zilla.runtime.binding.kafka@0.9.80/io.aklivity.zilla.runtime.binding.kafka.internal.stream.KafkaCacheBootstrapFactory$KafkaBootstrapFetchStream.onFetchReply(KafkaCacheBootstrapFactory.java:1514)
    at io.aklivity.zilla.runtime.engine@0.9.80/io.aklivity.zilla.runtime.engine.internal.registry.EngineWorker.handleDefaultReadReply(EngineWorker.java:1497)
    at io.aklivity.zilla.runtime.engine@0.9.80/io.aklivity.zilla.runtime.engine.internal.registry.EngineWorker.handleReadReply(EngineWorker.java:1428)
    at io.aklivity.zilla.runtime.engine@0.9.80/io.aklivity.zilla.runtime.engine.internal.registry.EngineWorker.handleRead(EngineWorker.java:1209)
    at io.aklivity.zilla.runtime.engine@0.9.80/io.aklivity.zilla.runtime.engine.internal.concurent.ManyToOneRingBuffer.read(ManyToOneRingBuffer.java:229)
    at io.aklivity.zilla.runtime.engine@0.9.80/io.aklivity.zilla.runtime.engine.internal.registry.EngineWorker.doWork(EngineWorker.java:817)
    ... 4 more
    Suppressed: java.lang.Exception: [engine/data#3]        [0x030300000000001c] streams=[consumeAt=0x00003930 (0x0000000000003930), produceAt=0x00003b60 (0x0000000000003b60)]
            at io.aklivity.zilla.runtime.engine@0.9.80/io.aklivity.zilla.runtime.engine.internal.registry.EngineWorker.doWork(EngineWorker.java:821)
            ... 4 more
```

--- a/runtime/binding-kafka/src/main/java/io/aklivity/zilla/runtime/binding/kafka/internal/stream/KafkaCacheBootstrapFactory.java
+++ b/runtime/binding-kafka/src/main/java/io/aklivity/zilla/runtime/binding/kafka/internal/stream/KafkaCacheBootstrapFactory.java
@@ -415,7 +415,7 @@ public final class KafkaCacheBootstrapFactory implements BindingHandler
             this.describeStream = new KafkaBootstrapDescribeStream(this);
             this.metaStream = new KafkaBootstrapMetaStream(this);
             this.fetchStreams = new ArrayList<>();
-            this.nextOffsetsById = new Long2LongHashMap(-1L);
+            this.nextOffsetsById = new Long2LongHashMap(-3L);
             this.defaultOffset = defaultOffset;
             this.consumers = new Int2ObjectHashMap<>();
             this.leadersByPartitionId = new Int2IntHashMap(-1);

--- a/runtime/binding-kafka/src/test/java/io/aklivity/zilla/runtime/binding/kafka/internal/stream/CacheBootstrapIT.java
+++ b/runtime/binding-kafka/src/test/java/io/aklivity/zilla/runtime/binding/kafka/internal/stream/CacheBootstrapIT.java
@@ -73,6 +73,23 @@ public class CacheBootstrapIT
     }
 
     @Test
+    @Configuration("cache.options.bootstrap.live.yaml")
+    @Specification({
+        "${app}/unmerged.fetch.message.values.live/server"})
+    public void shouldReceiveMergedMessageValuesLive() throws Exception
+    {
+        Thread.sleep(500);
+        k3po.start();
+        k3po.awaitBarrier("CHANGING_PARTITION_COUNT");
+        Thread.sleep(200); // allow A1, B1, A2, B2 to be merged
+        k3po.notifyBarrier("CHANGED_PARTITION_COUNT");
+        k3po.awaitBarrier("SENT_MESSAGE_A2");
+        k3po.awaitBarrier("SENT_MESSAGE_B2");
+        k3po.awaitBarrier("SENT_MESSAGE_C2");
+        k3po.finish();
+    }
+
+    @Test
     @Configuration("cache.yaml")
     @Specification({
         "${net}/group.fetch.message.value/client",

--- a/specs/binding-kafka.spec/src/main/scripts/io/aklivity/zilla/specs/binding/kafka/config/cache.options.bootstrap.live.yaml
+++ b/specs/binding-kafka.spec/src/main/scripts/io/aklivity/zilla/specs/binding/kafka/config/cache.options.bootstrap.live.yaml
@@ -1,0 +1,39 @@
+#
+# Copyright 2021-2023 Aklivity Inc.
+#
+# Aklivity licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+---
+name: test
+bindings:
+  app0:
+    type: kafka
+    kind: cache_client
+    routes:
+      - exit: cache0
+        when:
+          - topic: test
+  cache0:
+    type: kafka
+    kind: cache_server
+    options:
+      topics:
+        - name: test
+          defaultOffset: live
+      bootstrap:
+        - test
+    routes:
+      - exit: app1
+        when:
+          - topic: test

--- a/specs/binding-kafka.spec/src/main/scripts/io/aklivity/zilla/specs/binding/kafka/streams/application/merged/unmerged.fetch.message.values.live/client.rpt
+++ b/specs/binding-kafka.spec/src/main/scripts/io/aklivity/zilla/specs/binding/kafka/streams/application/merged/unmerged.fetch.message.values.live/client.rpt
@@ -1,0 +1,236 @@
+#
+# Copyright 2021-2023 Aklivity Inc.
+#
+# Aklivity licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+connect "zilla://streams/app1"
+    option zilla:window 8192
+    option zilla:transmission "half-duplex"
+
+write zilla:begin.ext ${kafka:beginEx()
+                               .typeId(zilla:id("kafka"))
+                               .describe()
+                                   .topic("test")
+                                   .config("cleanup.policy")
+                                   .config("max.message.bytes")
+                                   .config("segment.bytes")
+                                   .config("segment.index.bytes")
+                                   .config("segment.ms")
+                                   .config("retention.bytes")
+                                   .config("retention.ms")
+                                   .config("delete.retention.ms")
+                                   .config("min.compaction.lag.ms")
+                                   .config("max.compaction.lag.ms")
+                                   .config("min.cleanable.dirty.ratio")
+                                   .build()
+                               .build()}
+
+connected
+
+read zilla:begin.ext ${kafka:beginEx()
+                              .typeId(zilla:id("kafka"))
+                              .describe()
+                                  .topic("test")
+                                  .config("cleanup.policy")
+                                  .config("max.message.bytes")
+                                  .config("segment.bytes")
+                                  .config("segment.index.bytes")
+                                  .config("segment.ms")
+                                  .config("retention.bytes")
+                                  .config("retention.ms")
+                                  .config("delete.retention.ms")
+                                  .config("min.compaction.lag.ms")
+                                  .config("max.compaction.lag.ms")
+                                  .config("min.cleanable.dirty.ratio")
+                                  .build()
+                              .build()}
+
+read zilla:data.ext ${kafka:dataEx()
+                             .typeId(zilla:id("kafka"))
+                             .describe()
+                                 .config("cleanup.policy", "delete")
+                                 .config("max.message.bytes", 1000012)
+                                 .config("segment.bytes", 1073741824)
+                                 .config("segment.index.bytes", 10485760)
+                                 .config("segment.ms", 604800000)
+                                 .config("retention.bytes", -1)
+                                 .config("retention.ms", 604800000)
+                                 .config("delete.retention.ms", 86400000)
+                                 .config("min.compaction.lag.ms", 0)
+                                 .config("max.compaction.lag.ms", 9223372036854775807)
+                                 .config("min.cleanable.dirty.ratio", 0.5)
+                                 .build()
+                             .build()}
+
+read notify RECEIVED_CONFIG
+
+connect await RECEIVED_CONFIG
+        "zilla://streams/app1"
+    option zilla:window 8192
+    option zilla:transmission "half-duplex"
+
+write zilla:begin.ext ${kafka:beginEx()
+                               .typeId(zilla:id("kafka"))
+                               .meta()
+                                   .topic("test")
+                                   .build()
+                               .build()}
+
+connected
+
+read zilla:begin.ext ${kafka:beginEx()
+                              .typeId(zilla:id("kafka"))
+                              .meta()
+                                  .topic("test")
+                                  .build()
+                              .build()}
+
+read zilla:data.ext ${kafka:dataEx()
+                             .typeId(zilla:id("kafka"))
+                             .meta()
+                                 .partition(0, 1)
+                                 .partition(1, 2)
+                                 .build()
+                             .build()}
+read notify PARTITION_COUNT_2
+
+read zilla:data.ext ${kafka:dataEx()
+                             .typeId(zilla:id("kafka"))
+                             .meta()
+                                 .partition(0, 1)
+                                 .partition(1, 2)
+                                 .partition(2, 3)
+                                 .build()
+                             .build()}
+read notify PARTITION_COUNT_3
+
+connect await PARTITION_COUNT_2
+        "zilla://streams/app1"
+    option zilla:window 8192
+    option zilla:transmission "half-duplex"
+
+write zilla:begin.ext ${kafka:beginEx()
+                               .typeId(zilla:id("kafka"))
+                               .fetch()
+                                   .topic("test")
+                                   .partition(0, -1)
+                                   .build()
+                               .build()}
+
+connected
+
+read zilla:begin.ext ${kafka:beginEx()
+                              .typeId(zilla:id("kafka"))
+                              .fetch()
+                                  .topic("test")
+                                  .partition(0, 1, 2)
+                                  .build()
+                              .build()}
+
+read zilla:data.ext ${kafka:matchDataEx()
+                             .typeId(zilla:id("kafka"))
+                             .fetch()
+                                 .partition(0, 1, 2)
+                                 .build()
+                             .build()}
+read "Hello, world #A1"
+
+read zilla:data.ext ${kafka:matchDataEx()
+                             .typeId(zilla:id("kafka"))
+                             .fetch()
+                                 .partition(0, 2, 2)
+                                 .build()
+                             .build()}
+read "Hello, world #A2"
+read notify RECEIVED_MESSAGE_A2
+
+connect await PARTITION_COUNT_2
+        "zilla://streams/app1"
+    option zilla:window 8192
+    option zilla:transmission "half-duplex"
+
+write zilla:begin.ext ${kafka:beginEx()
+                               .typeId(zilla:id("kafka"))
+                               .fetch()
+                                   .topic("test")
+                                   .partition(1, -1)
+                                   .build()
+                               .build()}
+
+connected
+
+read zilla:begin.ext ${kafka:beginEx()
+                              .typeId(zilla:id("kafka"))
+                              .fetch()
+                                  .topic("test")
+                                  .partition(1, 1, 2)
+                                  .build()
+                              .build()}
+
+read zilla:data.ext ${kafka:matchDataEx()
+                             .typeId(zilla:id("kafka"))
+                             .fetch()
+                                 .partition(1, 1, 2)
+                                 .build()
+                             .build()}
+read "Hello, world #B1"
+
+read zilla:data.ext ${kafka:matchDataEx()
+                             .typeId(zilla:id("kafka"))
+                             .fetch()
+                                 .partition(1, 2, 2)
+                                 .build()
+                             .build()}
+read "Hello, world #B2"
+read notify RECEIVED_MESSAGE_B2
+
+connect await PARTITION_COUNT_3
+        "zilla://streams/app1"
+    option zilla:window 8192
+    option zilla:transmission "half-duplex"
+
+write zilla:begin.ext ${kafka:beginEx()
+                               .typeId(zilla:id("kafka"))
+                               .fetch()
+                                   .topic("test")
+                                   .partition(2, -1)
+                                   .build()
+                               .build()}
+
+connected
+
+read zilla:begin.ext ${kafka:beginEx()
+                              .typeId(zilla:id("kafka"))
+                              .fetch()
+                                  .topic("test")
+                                  .partition(2, 1, 2)
+                                  .build()
+                              .build()}
+
+read zilla:data.ext ${kafka:matchDataEx()
+                             .typeId(zilla:id("kafka"))
+                             .fetch()
+                                 .partition(2, 1, 2)
+                                 .build()
+                             .build()}
+read "Hello, world #C1"
+
+read zilla:data.ext ${kafka:matchDataEx()
+                             .typeId(zilla:id("kafka"))
+                             .fetch()
+                                 .partition(2, 2, 2)
+                                 .build()
+                             .build()}
+read "Hello, world #C2"
+read notify RECEIVED_MESSAGE_C2

--- a/specs/binding-kafka.spec/src/main/scripts/io/aklivity/zilla/specs/binding/kafka/streams/application/merged/unmerged.fetch.message.values.live/server.rpt
+++ b/specs/binding-kafka.spec/src/main/scripts/io/aklivity/zilla/specs/binding/kafka/streams/application/merged/unmerged.fetch.message.values.live/server.rpt
@@ -1,0 +1,251 @@
+#
+# Copyright 2021-2023 Aklivity Inc.
+#
+# Aklivity licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+property deltaMillis 0L
+property newTimestamp ${kafka:timestamp() + deltaMillis}
+
+accept "zilla://streams/app1"
+    option zilla:window 8192
+    option zilla:transmission "half-duplex"
+
+accepted
+
+read zilla:begin.ext ${kafka:beginEx()
+                              .typeId(zilla:id("kafka"))
+                              .describe()
+                                  .topic("test")
+                                  .config("cleanup.policy")
+                                  .config("max.message.bytes")
+                                  .config("segment.bytes")
+                                  .config("segment.index.bytes")
+                                  .config("segment.ms")
+                                  .config("retention.bytes")
+                                  .config("retention.ms")
+                                  .config("delete.retention.ms")
+                                  .config("min.compaction.lag.ms")
+                                  .config("max.compaction.lag.ms")
+                                  .config("min.cleanable.dirty.ratio")
+                                  .build()
+                              .build()}
+
+connected
+
+write zilla:begin.ext ${kafka:beginEx()
+                               .typeId(zilla:id("kafka"))
+                               .describe()
+                                   .topic("test")
+                                   .config("cleanup.policy")
+                                   .config("max.message.bytes")
+                                   .config("segment.bytes")
+                                   .config("segment.index.bytes")
+                                   .config("segment.ms")
+                                   .config("retention.bytes")
+                                   .config("retention.ms")
+                                   .config("delete.retention.ms")
+                                   .config("min.compaction.lag.ms")
+                                   .config("max.compaction.lag.ms")
+                                   .config("min.cleanable.dirty.ratio")
+                                   .build()
+                               .build()}
+write flush
+
+write zilla:data.ext ${kafka:dataEx()
+                              .typeId(zilla:id("kafka"))
+                              .describe()
+                                  .config("cleanup.policy", "delete")
+                                  .config("max.message.bytes", 1000012)
+                                  .config("segment.bytes", 1073741824)
+                                  .config("segment.index.bytes", 10485760)
+                                  .config("segment.ms", 604800000)
+                                  .config("retention.bytes", -1)
+                                  .config("retention.ms", 604800000)
+                                  .config("delete.retention.ms", 86400000)
+                                  .config("min.compaction.lag.ms", 0)
+                                  .config("max.compaction.lag.ms", 9223372036854775807)
+                                  .config("min.cleanable.dirty.ratio", 0.5)
+                                  .build()
+                              .build()}
+write flush
+
+accepted
+
+read zilla:begin.ext ${kafka:beginEx()
+                              .typeId(zilla:id("kafka"))
+                              .meta()
+                                  .topic("test")
+                                  .build()
+                              .build()}
+
+connected
+
+write zilla:begin.ext ${kafka:beginEx()
+                               .typeId(zilla:id("kafka"))
+                               .meta()
+                                   .topic("test")
+                                   .build()
+                               .build()}
+write flush
+
+write zilla:data.ext ${kafka:dataEx()
+                              .typeId(zilla:id("kafka"))
+                              .meta()
+                                  .partition(0, 1)
+                                  .partition(1, 2)
+                                  .build()
+                              .build()}
+write flush
+
+write await SENT_MESSAGE_A2
+write await SENT_MESSAGE_B2
+
+write notify CHANGING_PARTITION_COUNT
+write await CHANGED_PARTITION_COUNT
+
+write zilla:data.ext ${kafka:dataEx()
+                              .typeId(zilla:id("kafka"))
+                              .meta()
+                                  .partition(0, 1)
+                                  .partition(1, 2)
+                                  .partition(2, 3)
+                                  .build()
+                              .build()}
+write flush
+
+accepted
+
+read zilla:begin.ext ${kafka:beginEx()
+                              .typeId(zilla:id("kafka"))
+                              .fetch()
+                                  .topic("test")
+                                  .partition(0, -1)
+                                  .build()
+                              .build()}
+
+connected
+
+write zilla:begin.ext ${kafka:beginEx()
+                               .typeId(zilla:id("kafka"))
+                               .fetch()
+                                   .topic("test")
+                                   .partition(0, 1, 2)
+                                   .build()
+                               .build()}
+write flush
+
+write zilla:data.ext ${kafka:dataEx()
+                              .typeId(zilla:id("kafka"))
+                              .fetch()
+                                  .timestamp(newTimestamp)
+                                  .partition(0, 1, 2)
+                                  .build()
+                              .build()}
+write "Hello, world #A1"
+write flush
+
+write zilla:data.ext ${kafka:dataEx()
+                              .typeId(zilla:id("kafka"))
+                              .fetch()
+                                  .timestamp(newTimestamp)
+                                  .partition(0, 2, 2)
+                                  .build()
+                              .build()}
+write "Hello, world #A2"
+write flush
+write notify SENT_MESSAGE_A2
+
+accepted
+
+read zilla:begin.ext ${kafka:beginEx()
+                              .typeId(zilla:id("kafka"))
+                              .fetch()
+                                  .topic("test")
+                                  .partition(1, -1)
+                                  .build()
+                              .build()}
+
+connected
+
+write zilla:begin.ext ${kafka:beginEx()
+                               .typeId(zilla:id("kafka"))
+                               .fetch()
+                                   .topic("test")
+                                   .partition(1, 1, 2)
+                                   .build()
+                               .build()}
+write flush
+
+write zilla:data.ext ${kafka:dataEx()
+                              .typeId(zilla:id("kafka"))
+                              .fetch()
+                                  .timestamp(newTimestamp)
+                                  .partition(1, 1, 2)
+                                  .build()
+                              .build()}
+write "Hello, world #B1"
+write flush
+
+write zilla:data.ext ${kafka:dataEx()
+                              .typeId(zilla:id("kafka"))
+                              .fetch()
+                                  .timestamp(newTimestamp)
+                                  .partition(1, 2, 2)
+                                  .build()
+                              .build()}
+write "Hello, world #B2"
+write flush
+write notify SENT_MESSAGE_B2
+
+accepted
+
+read zilla:begin.ext ${kafka:beginEx()
+                              .typeId(zilla:id("kafka"))
+                              .fetch()
+                                  .topic("test")
+                                  .partition(2, -1)
+                                  .build()
+                              .build()}
+
+connected
+
+write zilla:begin.ext ${kafka:beginEx()
+                               .typeId(zilla:id("kafka"))
+                               .fetch()
+                                   .topic("test")
+                                   .partition(2, 1, 2)
+                                   .build()
+                               .build()}
+write flush
+
+write zilla:data.ext ${kafka:dataEx()
+                              .typeId(zilla:id("kafka"))
+                              .fetch()
+                                  .timestamp(newTimestamp)
+                                  .partition(2, 1, 2)
+                                  .build()
+                              .build()}
+write "Hello, world #C1"
+write flush
+
+write zilla:data.ext ${kafka:dataEx()
+                              .typeId(zilla:id("kafka"))
+                              .fetch()
+                                  .timestamp(newTimestamp)
+                                  .partition(2, 2, 2)
+                                  .build()
+                              .build()}
+write "Hello, world #C2"
+write flush
+write notify SENT_MESSAGE_C2

--- a/specs/binding-kafka.spec/src/test/java/io/aklivity/zilla/specs/binding/kafka/streams/application/BootstrapIT.java
+++ b/specs/binding-kafka.spec/src/test/java/io/aklivity/zilla/specs/binding/kafka/streams/application/BootstrapIT.java
@@ -45,7 +45,6 @@ public class BootstrapIT
         k3po.finish();
     }
 
-
     @Test
     @Specification({
         "${app}/group.fetch.message.value/client",

--- a/specs/binding-kafka.spec/src/test/java/io/aklivity/zilla/specs/binding/kafka/streams/application/MergedIT.java
+++ b/specs/binding-kafka.spec/src/test/java/io/aklivity/zilla/specs/binding/kafka/streams/application/MergedIT.java
@@ -497,6 +497,18 @@ public class MergedIT
 
     @Test
     @Specification({
+        "${app}/unmerged.fetch.message.values.live/client",
+        "${app}/unmerged.fetch.message.values.live/server"})
+    public void shouldFetchUnmergedMessageValuesLive() throws Exception
+    {
+        k3po.start();
+        k3po.awaitBarrier("CHANGING_PARTITION_COUNT");
+        k3po.notifyBarrier("CHANGED_PARTITION_COUNT");
+        k3po.finish();
+    }
+
+    @Test
+    @Specification({
         "${app}/unmerged.fetch.partition.offsets.latest/client",
         "${app}/unmerged.fetch.partition.offsets.latest/server"})
     public void shouldFetchUnmergedPartitionOffsetsLatest() throws Exception

